pair-programming/pair-programming.md

```diff
@@ -62,5 +57,8 @@ Pair programming is a very common way to write code and to solve software engine
 
 1. Techtonica staff will assign pairs. Ask and answer the questions we came up with during the Group Practice. Work with your pair partner on [this quick code challenge on codepen](https://codepen.io/ulfiw/pen/YQoQgx).  Be sure to fork it first.  
 
+### Suplementary Materials
```

Comment:
This new section is not necessary.  Please put the 3 min video back in Lesson or Materials and make it required.

Reply:
Done @alodahl